### ISSUE TITLE
Automatically image

### ISSUE THREAD

**Issue by robsonolivesa** [dularion/streama#194]
Is possible the image of a manual video automatically be the first frame of the video?
Thanks

**Comment by dularion** [comment]
Yeah I've been thinking about that too. While the video is loading, right?

**Comment by robsonolivesa** [comment]
Yes! I will try to develop this feature on my fork.

**Comment by dularion** [comment]
Cool! Don't forget to PR :)

**Comment by robsonolivesa** [comment]
Ok, thanks and congratulations for this great project.
